include/gsl/gsl

```diff
@@ -109,6 +109,13 @@ private:
     T ptr_;
 };
 
+template <class T>
```

Comment:
oppsss... ha. I have that in my patch and forgot to port it to this one. Sorry about that